widgetssdk/src/main/java/com/glia/widgets/survey/SurveyController.java

```diff
@@ -111,6 +136,7 @@ public void onAnswer(@NonNull Survey.Answer answer) {
     public void onCancelClicked() {
         if (view != null) {
             view.finish();
+            controllerFactory.destroySurveyController();
```

Comment:
could reset the controller here instead of destroy in controller factory

Reply:
Used `resetController()` instead of `destroySurveyController()`.